barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -115,13 +117,36 @@ class GoblinProver {
      */
     void merge(MegaCircuitBuilder& circuit_builder)
```

Comment:
when is this function used?

Reply:
in client ivc I guess? is the plan that aztec ivc replaces client ivc at some point?

Reply:
Yeah the plan is definitely that AztecIvc replaces ClientIvc as the IVC scheme used for aztec but it's not totally clear whether ClientIvc should go away or not. It might still have some use as a more general IVC scheme for noir or something